src/registrar/views/domain_request.py

```diff
@@ -413,14 +476,19 @@ def get_context_data(self):
                 "user": self.request.user,
                 "requested_domain__name": requested_domain_name,
             }
+
+        # Hides the requests and domains buttons in the navbar
+        context_stuff["hide_requests"] = self.is_portfolio
+        context_stuff["hide_domains"] = self.is_portfolio
```

Comment:
Is there a reason these are two separate keys? As of now it looks like they'd always be the same.

Reply:
I don't mind them being separated, I just wanted to know if I was missing something. It's fine to leave as-is IMO.